community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStore.java

```diff
@@ -967,26 +984,5 @@ public void rebuildCountStoreIfNeeded() throws IOException
     {
         // TODO: move this to LifeCycle
         counts.start();
```

Comment:
Yes, this method could use a better name... Although it actually still does what the name says, only the logic is not directly visible in this method.